src/app/config.rs

```diff
@@ -1,3 +1,12 @@
+use super::control::ControlAddr;
+use super::identity;
+use crate::addr;
```

Comment:
Can/should this be
```rust
use crate::{
    addr,
    convert::TryFrom,
    ... 
};
```

Reply:
it can be but, since this isn't something that is enforced by rustfmt, i don't particularly care one way or the other.

Reply:
fair; i only brought it up due to your comment about consolidating `use` statements, which i realise i kind of misread to imply that you tried to consolidate individual `use`s into blocks.